
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  </head>
<body ng-app="app">
<div class="container">
        <h2>Metascape Gene List Analysis Report</h2><p/>  <a href="http://metascape.org">metascape.org</a><sup>1</sup><p/>

              <h3>Bar Graph Summary</h3>
              <div class="panel panel-info">
                <div class="panel-heading">Figure 1. Bar graph of enriched terms across input gene lists, colored by p-values.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_heatmap/HeatmapSelectedGO.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_heatmap/HeatmapSelectedGO.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    
                <tr><td>The top-level Gene Ontology biological processes can be viewed here.<br>
                <a href="./Enrichment_heatmap/HeatmapSelectedGOParent.png"><img src="./Enrichment_heatmap/HeatmapSelectedGOParent.png" style="height:auto; width:auto; max-height:100px;"></a>
                <a href='./Enrichment_heatmap/HeatmapSelectedGOParent.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a></td></tr>
                
                </table>
                </div>
              </div><p/>
      <h3>Gene Lists</h3>
      User-provided gene identifiers are first converted into their corresponding H. sapiens Entrez gene IDs using the latest version of the database (last updated on 2022-01-01).  If multiple identifiers correspond to the same Entrez gene ID, they will be considered as a single Entrez gene ID in downstream analyses.   The gene lists are summarized in Table 1.<p/>
          <div class="panel panel-info">
            <div class="panel-heading">Table 1. Statistics of input gene lists.</div>
            <div class="panel-body"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">Name</TH>
      <TH class="info">Total</TH>
      <TH class="info">Unique</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>MyList</TD>
      <TD>20</TD>
      <TD>20</TD>
    </TR>
  </TBODY>
</TABLE></div>
          </div>


              <h3>Gene Annotation</h3>
              The following are the list of annotations retrieved from the latest version of the database (last updated on 2022-01-01) (Table 2).<p/>
              <div class="panel panel-info">
                <div class="panel-heading">Table 2. Gene annotations extracted</div>
                <div class="panel-body"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">Name</TH>
      <TH class="info">Type</TH>
      <TH class="info">Description</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>Gene Symbol</TD>
      <TD>Description</TD>
      <TD>Primary HUGO gene symbol.</TD>
    </TR>
    <TR>
      <TD>Description</TD>
      <TD>Description</TD>
      <TD>Short description.</TD>
    </TR>
    <TR>
      <TD>Biological Process (GO)</TD>
      <TD>Function/Location</TD>
      <TD>Descriptions summarized based on gene ontology database, where up to three most informative GO terms are kept.</TD>
    </TR>
    <TR>
      <TD>Kinase Class (UniProt)</TD>
      <TD>Function/Location</TD>
      <TD>Detailed kinase classes.</TD>
    </TR>
    <TR>
      <TD>Protein Function (Protein Atlas)</TD>
      <TD>Function/Location</TD>
      <TD>Protein Function (Protein Atlas)</TD>
    </TR>
    <TR>
      <TD>Subcellular Location (Protein Atlas)</TD>
      <TD>Function/Location</TD>
      <TD>Sucellular Location (Protein Atlas)</TD>
    </TR>
    <TR>
      <TD>Drug (DrugBank)</TD>
      <TD>Genotype/Phenotype/Disease</TD>
      <TD>Drug information for the given gene as target.</TD>
    </TR>
    <TR>
      <TD>Canonical Pathways
</TD>
      <TD>Ontology</TD>
      <TD>Canonical Pathways
</TD>
    </TR>
    <TR>
      <TD>Hallmark Gene Sets
</TD>
      <TD>Ontology</TD>
      <TD>Hallmark Gene Sets
</TD>
    </TR>
  </TBODY>
</TABLE></div>
              </div><p/>


              <h3>Pathway and Process Enrichment Analysis</h3>
              For each given gene list, pathway and process enrichment analysis has been carried out with the following ontology sources: KEGG Pathway, GO Biological Processes, Reactome Gene Sets, Canonical Pathways, Cell Type Signatures, CORUM, TRRUST, DisGeNET, PaGenBase, Transcription Factor Targets, WikiPathways and COVID.  All genes in the genome have been used as the enrichment background. Terms with a p-value &lt; 0.01, a minimum count of 3, and an enrichment factor &gt; 1.5 (the enrichment factor is the ratio between the observed counts and the counts expected by chance) are collected and grouped into clusters based on their membership similarities.  More specifically, p-values are calculated based on the accumulative hypergeometric distribution<sup>2</sup>, and q-values are calculated using the Benjamini-Hochberg procedure to account for multiple testings<sup>3</sup>.  Kappa scores<sup>4</sup> are used as the similarity metric when performing hierachical clustering on the enriched terms, and sub-trees with a similarity of > 0.3 are considered a cluster.  The most statistically significant term within a cluster is chosen to represent the cluster.<p/>
          <div class="panel panel-info">
          <div class="panel-heading">Table 3. Top 4 clusters with their representative enriched terms (one per cluster).  "Count" is the number of genes in the user-provided lists with membership in the given ontology term. "%" is the percentage of all of the user-provided genes that are found in the given ontology term (only input genes with at least one ontology term annotation are included in the calculation).  "Log10(P)" is the p-value in log base 10.  "Log10(q)" is the multi-test adjusted p-value in log base 10.</div>
                <div class="panel-body"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Category</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>GO:0032787</TD>
      <TD>GO Biological Processes</TD>
      <TD>monocarboxylic acid metabolic process</TD>
      <TD>4</TD>
      <TD>21.05</TD>
      <TD>-3.66</TD>
      <TD>0.00</TD>
    </TR>
    <TR>
      <TD>GO:0016050</TD>
      <TD>GO Biological Processes</TD>
      <TD>vesicle organization</TD>
      <TD>3</TD>
      <TD>15.79</TD>
      <TD>-3.17</TD>
      <TD>0.00</TD>
    </TR>
    <TR>
      <TD>GO:0007268</TD>
      <TD>GO Biological Processes</TD>
      <TD>chemical synaptic transmission</TD>
      <TD>3</TD>
      <TD>15.79</TD>
      <TD>-2.73</TD>
      <TD>0.00</TD>
    </TR>
    <TR>
      <TD>GO:0045859</TD>
      <TD>GO Biological Processes</TD>
      <TD>regulation of protein kinase activity</TD>
      <TD>3</TD>
      <TD>15.79</TD>
      <TD>-2.14</TD>
      <TD>0.00</TD>
    </TR>
  </TBODY>
</TABLE></div>
              </div><p/>
                  To further capture the relationships between the terms, a subset of enriched terms have been selected and rendered as a network plot, where terms with a similarity &gt; 0.3 are connected by edges.  We select the terms with the best p-values from each of the 20 clusters, with the constraint that there are no more than 15 terms per cluster and no more than 250 terms in total.  The network is visualized using <a href="http://www.cytoscape.org">Cytoscape</a><sup>5</sup>, where each node represents an enriched term and is colored first by its cluster ID (Figure 2.a) and then by its p-value (Figure 2.b).  These networks can be interactively viewed in Cytoscape through the .cys files (contained in the Zip package, which also contains a publication-quality version as a PDF) or within a browser by clicking on the web icon.  For clarity, term labels are only shown for one term per cluster, so it is recommended to use Cytoscape or a browser to visualize the network in order to inspect all node labels.  We can also export the network into a PDF file within Cytoscape, and then edit the labels using Adobe Illustrator for publication purposes.  To switch off all labels, delete the "Label" mapping under the "Style" tab within Cytoscape, and then export the network view.<p/>
                  <div class="panel panel-info">
                    <div class="panel-heading">Figure 2. Network of enriched terms: (a) colored by cluster ID, where nodes that share the same cluster ID are typically close to each other; (b) colored by p-value, where terms containing more genes tend to have a more significant p-value.</div>
                    <div class="panel-body">
                    <table>
                        <tr>
                            <td>
                                <img src="./Enrichment_GO/ColorByCluster.png" style="width:500px;">
                            </td>
                            <td>
                                <img src="./Enrichment_GO/ColorByPValue.png" style="width:500px;">
                            </td>
                        </tr>
                        <tr>
                            <td align='center'>
                                <a href='./Enrichment_GO/ColorByCluster.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_GO/GONetwork.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_GO/GONetwork.html?Network=GONetwork&Style=ColorByCluster' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    
                            </td>
                            <td align='center'>
                                <a href='./Enrichment_GO/ColorByPValue.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_GO/GONetwork.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_GO/GONetwork.html?Network=GONetwork&Style=ColorByPValue' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    
                            </td>
                        </tr>
                    </table>
                    </div>

                  </div><p/>
<h3>Protein-protein Interaction Enrichment Analysis</h3>
For each given gene list, protein-protein interaction enrichment analysis has been carried out with the following databases: STRING<sup>6</sup>, BioGrid<sup>7</sup>, OmniPath<sup>8</sup>, InWeb_IM<sup>9</sup>.Only physical interactions in STRING (physical score &gt 0.132) and BioGrid are used (<a href="http://metascape.org/blog/?p=219">details</a>).  The resultant network contains the subset of proteins that form physical interactions with at least one other member in the list.  If the network contains between 3 and 500 proteins, the Molecular Complex Detection (MCODE) algorithm<sup>10</sup> has been applied to identify densely connected network components.  The MCODE networks identified for individual gene lists have been gathered and are shown in Figure 3.<p/>

                    Pathway and process enrichment analysis has been applied to each MCODE component independently, and the three best-scoring terms by p-value have been retained as the functional description of the corresponding components, shown in the tables underneath corresponding network plots within Figure 3.<p/>

      <div class="panel panel-info">
        <div class="panel-heading">Figure 3. Protein-protein interaction network and MCODE components identified in the gene lists.</div>
        <div class="panel-body">
        
                <table border-collapse='collapse'>
                <tr><td><img src="./Enrichment_PPI/MyList_PPIColorByCluster.png" style="width:500px;"></td></tr>
                <tr><td align="center"><a href='./Enrichment_PPI/MyList_PPIColorByCluster.pdf' title='download PDF file'>
                        <img class='link' src='icon/PDF48.png' >
                        </a>
                     
                        <a href='./Enrichment_PPI/MCODE_PPI.cys' title='download CYS file'>
                        <img class='link' src='icon/CYS48.png' >
                        </a>
                    
                        
                    
                        
                        <a href='Enrichment_PPI/PPINetwork.html?Network=MyList_PPIColorByCluster&Style=PPIColorByClusterNoLabel&isPPI=True' title='interactive cytoscape' target='_blank'>
                        <img class='link' src='icon/WEB_CYS48.png' >
                        </a>
                        
                    </td></tr>
                <tr><td align="center" valign="top"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Log10(P)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>GO:0016050</TD>
      <TD>vesicle organization</TD>
      <TD>-4.8</TD>
    </TR>
    <TR>
      <TD>GO:0061024</TD>
      <TD>membrane organization</TD>
      <TD>-3.5</TD>
    </TR>
  </TBODY>
</TABLE></td></tr>
                </table>
        </div>
      </div><p/>

<h3>Quality Control and Association Analysis</h3>

<p>Gene list enrichments are identified in the following ontology categories: COVID, Transcription_Factor_Targets. All genes in the genome have been used as the enrichment background. Terms with a p-value &lt; 0.01, a minimum count of 3, and an enrichment factor &gt; 1.5 (the enrichment factor is the ratio between the observed counts and the counts expected by chance) are collected and grouped into clusters based on their membership similarities.  The top few enriched clusters (one term per cluster) are shown in the Figure 4-5.  The algorithm used here is the same as that is used for pathway and process enrichment analysis.<p/>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 4. Summary of enrichment analysis in COVID<sup>11</sup>.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_COVID.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_COVID.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD><a href="https://metascape.org/COVID#200_COVID051" target="_TERM">COVID051</a></TD>
      <TD>RNA_Xiong_BALF_Down</TD>
      <TD>3</TD>
      <TD>16</TD>
      <TD>-3.10</TD>
      <TD>0</TD>
    </TR>
    <TR>
      <TD><a href="https://metascape.org/COVID#200_COVID385" target="_TERM">COVID385</a></TD>
      <TD>Interactome_Laurent_HEK293_24h_ORF7A</TD>
      <TD>3</TD>
      <TD>16</TD>
      <TD>-3.10</TD>
      <TD>0</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>

              <div class="panel panel-info">
                <div class="panel-heading">Figure 5. Summary of enrichment analysis in Transcription Factor Targets<sup>12</sup>.</div>
                <div class="panel-body">

                <table>
                    <tr>
                        <td>
                            <img src="./Enrichment_QC/HeatmapSelectedGO_Transcription_Factor_Targets.png" style="width:1000px;">
                        </td>
                    </tr>
                    <tr>
                        <td align='center'>
                            
                <a href='./Enrichment_QC/HeatmapSelectedGO_Transcription_Factor_Targets.pdf' title='download PDF file'>
                <img class='link' src='icon/PDF48.png' >
                </a>
                
                        </td>
                    </tr>
                    <tr>
                        <td>
                            <br><br><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">GO</TH>
      <TH class="info">Description</TH>
      <TH class="info">Count</TH>
      <TH class="info">%</TH>
      <TH class="info">Log10(P)</TH>
      <TH class="info">Log10(q)</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M15556" target="_TERM">M15556</a></TD>
      <TD>FREAC7 01</TD>
      <TD>3</TD>
      <TD>16</TD>
      <TD>-3.60</TD>
      <TD>0</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M17000" target="_TERM">M17000</a></TD>
      <TD>CP2 01</TD>
      <TD>3</TD>
      <TD>16</TD>
      <TD>-3.30</TD>
      <TD>0</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M13889" target="_TERM">M13889</a></TD>
      <TD>PPAR DR1 Q2</TD>
      <TD>3</TD>
      <TD>16</TD>
      <TD>-3.20</TD>
      <TD>0</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M30042" target="_TERM">M30042</a></TD>
      <TD>KMT2D TARGET GENES</TD>
      <TD>4</TD>
      <TD>21</TD>
      <TD>-2.90</TD>
      <TD>0</TD>
    </TR>
    <TR>
      <TD><a href="https://www.gsea-msigdb.org/gsea/msigdb/geneset_page.jsp?systematicName=M9645" target="_TERM">M9645</a></TD>
      <TD>GGGYGTGNY UNKNOWN</TD>
      <TD>3</TD>
      <TD>16</TD>
      <TD>-2.10</TD>
      <TD>0</TD>
    </TR>
  </TBODY>
</TABLE>
                        </td>
                    </tr>
                </table>

                </div>
              </div>
    <h3>Reference</h3></p>
    <ol style="list-style: decimal inside;">
      <li>Zhou et al., Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Nature Communications (2019) 10(1):1523.</li>
      <li>Zar, J.H. Biostatistical Analysis 1999 4th edn., NJ Prentice Hall, pp. 523</li>
      <li>Hochberg Y., Benjamini Y. More powerful procedures for multiple significance testing. Statistics in Medicine (1990) 9:811-818.</li>
      <li>Cohen, J. A coefficient of agreement for nominal scales. Educ. Psychol. Meas. (1960) 20:27-46.</li>
      <li>Shannon P. et al., Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res (2003) 11:2498-2504.</li>
      <li>Szklarczyk D. et al. STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. (2019) 47:D607-613.</li>
      <li>Stark C. et al. BioGRID: a general repository for interaction datasets. Nucleic Acids Res. (2006) 34:D535-539.</li>
      <li>Turei D. et al. A scored human protein-protein interaction network to catalyze genomic interpretation. Nat. Methods. (2016) 13:966-967.</li>
      <li>Li T. et al. A scored human protein-protein interaction network to catalyze genomic interpretation. Nat. Methods. (2017) 14:61-64.</li>
      <li>Bader, G.D. et al. An automated method for finding molecular complexes in large protein interaction networks. BMC bioinformatics (2003) 4:2.</li>
      <li>https://metascape.org/COVID.</li>
      <li>Subramanian A, et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A 102, 15545-15550 (2005).</li>
</ol>
</div>

</body></html>